docs/introduction/resources.md

```diff
@@ -22,6 +22,15 @@ is a model. Your library may have two copies that you can check out. Those are r
 
 *Names TBD.
```

Comment:
Is this a unclosed italic? Should it have a `*` at the end as well? Everything after is appearing as italics :)